dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeConfig.java

```diff
@@ -274,6 +281,27 @@ private static Long getLongValue(Map<String, Object> configProps, String propert
 		return (Long) value;
 	}
 	
+	/**
+	 * Get a {@link Integer} value from the config properties.
+	 * 
+	 * @param configProps
+	 *            The configuration properties passed in by declarative
+	 *            services.
+	 * @param property
+	 *            The property to lookup.
+	 * @param outcomeOnNull
+	 *            The value to assign if the lookup is null.
+	 * @return The {@link Integer} value, or outcomeOnNull if it doesn't exist.
+	 */
+	@Trivial
+	private static Integer getIntValue(Map<String, Object> configProps, String property, Integer outcomeOnNull) {
```

Comment:
If you change the types in the metatype to `ibm:type="duration"` then these will need no longer be needed.

Reply:
Removed